nikola/nikola.py

```diff
@@ -1352,129 +1352,14 @@ def flatten(task):
             'task_dep': task_dep
         }
 
-    def scan_posts(self, really=False, ignore_quit=False, quiet=False):
```

Comment:
The parameters are useful (for example, coil depends on `really`) — you should leave them in and make them usable in the plugin.